### ISSUE TITLE
Relative path should not report as absolute path

### ISSUE THREAD

**Issue by chucklu** [SonarSource/sonar-dotnet#2425]
### Description

The following code should not be recognized as absolute path
```C#
  Server.Transfer("~/Pages/System/Error/Error.aspx", false);
```
[HttpServerUtility.Transfer Method](https://docs.microsoft.com/en-us/dotnet/api/system.web.httpserverutility.transfer?view=netframework-4.8#System_Web_HttpServerUtility_Transfer_System_String_)

### Repro steps

use the code provided above

### Expected behavior

The path used with ~ is relative path, should not report “URIs should not be hardcoded”

### Actual behavior

refactor your code not to use hardcoded absolute paths or uris
![image](https://user-images.githubusercontent.com/3004057/57744095-4dc26d80-76fa-11e9-80d8-7d40787d6f6f.png)

### Known workarounds

Please provide a description of any known workarounds.

### Related information

* SonarC# Version
7.9.1
![image](https://user-images.githubusercontent.com/3004057/57744110-66cb1e80-76fa-11e9-802c-42978f49d987.png)

* Visual Studio Version
vs2019 community

* If running through the Scanner for MSBuild, its version
sonar-scanner-msbuild-4.3.1.1372-net46

By the way, I find you have fixed a similar issue https://github.com/SonarSource/sonar-dotnet/issues/1586

**Comment by andrei-epure-sonarsource** [comment]
Hi, this got fixed in #2371 which got into the 7.13 release. Please update to the latest version to benefit of the fix.

**Comment by chucklu** [comment]
Thanks
